pomcorn/element.py

```diff
@@ -15,7 +15,7 @@
     from pomcorn.web_view import WebView
 
 
-class Element(Generic[locators.TLocator]):
+class _Element(Generic[locators.TLocator]):
```

Comment:
Maybe one of `LocatedElement`, `ElementWithLocator`, `PomcornElement`, `TLocatorElement`, `GenericElement` ?